tests/phpunit/Mutator/MutatorResolverTest.php

```diff
@@ -330,6 +334,107 @@ public function test_it_cannot_resolve_an_unknown_mutator(): void
         }
     }
 
+    public function test_it_correct_when_profile_overrides_mutator(): void
+    {
+        $resolvedMutators = $this->mutatorResolver->resolve([
+            'DecrementInteger' => [
+                'ignore' => [
+                    'Infected\\SourceClass::add',
+                ],
+            ],
+            '@number' => true,
+        ]);
+
+        $this->assertSame(
+            [
```

Comment:
I thought about more generic way to build expected result otherwise we will need to fix this test each time when we add new mutator to the `@number` profile (do not expect that it may happen often).

Reply:
Yeah, it should be possible to make this test a little less brittle, but it'll cost in visibility which I think isn't worth it right now.